openthread_border_router/translations/en.yaml

```diff
@@ -10,6 +10,11 @@ configuration:
   flow_control:
     name: Hardware flow control
     description: Enable hardware flow control for serial port.
+  network_device:
+    name: Network Device
+    description: >-
+      <host:port> when connecting to a device via sockets (takes precedence
+      over above configuration). Not suitable for Wifi connections.
```

Comment:
```suggestion
      over above configuration). Not suitable for WiFi or other high latency
      connections.
```

Reply:
I really hope people don't try run this over the internet! ;)